lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialServiceTest.java

```diff
@@ -132,7 +134,12 @@ void shouldReturnAVerifiedCredentialWithFailScoreOnNotAuthorised()
                 new VerifiableCredentialService(
                         signedJwtFactory, mockConfigurationService, mockObjectMapper);
         when(mockObjectMapper.convertValue(any(Evidence.class), eq(Map.class)))
-                .thenReturn(Map.of("verificationScore", 0));
+                .thenReturn(
```

Comment:
Let's leave this unit test as is
& add a new unit test to for this specific scenario where there are sufficient questions but insufficient correct answers